Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -824,7 +824,7 @@ bool MainWindow::RequestStop()
       Core::SetState(Core::State::Paused);
 
     auto confirm = ModalMessageBox::question(
-        this, tr("Confirm"),
+        m_rendering_to_main ? static_cast<QWidget*>(this) : m_render_widget, tr("Confirm"),
```

Comment:
Why is this cast needed? Does the ternary operator not compile otherwise?

Reply:
clang gives me the following error:
```
../Source/Core/DolphinQt/MainWindow.cpp:827:29: error: incompatible operand types ('MainWindow *' and 'RenderWidget *')
        m_rendering_to_main ? this : m_render_widget, tr("Confirm"),
```

Reply:
Ah, right, it works when you have one pointer that's a QWidget* and one that derives from QWidget*, but not when you have two different types that derive from QWidget*. Makes sense then. I think I was assuming that `m_render_widget` already had the type QWidget*.